assets/locales/en.json

```
@@ -42,5 +42,11 @@
   "runningLatestVersion": "You seem to be running the latest version.",
   "couldNotCheckForUpdates": "Could not check for updates",
   "pleaseCheckInternet": "Please check your Internet connection.",
-  "checkForUpdates": "Check for Updates..."
+  "checkForUpdates": "Check for Updates...",
+  "yes": "Yes",
+  "no": "No",
+  "anErrorHasOccurred": "An error has occurred",
+  "anUnexpectedErrorHasOccurred": "An unexpected error has occurred and it does not prevent the execution of IPFS Desktop. You can either inspect the logs for yourself or report the error to the developers.",
```

We can simplify the language here but my intuition is "inspect the logs"  is almost never a useful action for anyone other than the developers of the app. 

```suggestion
  "anUnexpectedErrorHasOccurred": "An error occurred but it does not prevent the execution of IPFS Desktop. You can either inspect the logs for yourself or report the error to the developers.",
```

Perhaps this kind of error could be logged to analytics if enabled?

---

Perhaps, we can think about that after this PR is finished.

What would you replace the inspect the logs part for?